peachjam/graph/ranker.py

```diff
@@ -102,7 +71,7 @@ def bulk_update(self, works):
             f"updating index with {len(docs)} documents",
         )
         searchable_doc = SearchableDocument()
-        actions = [
+        actions = (
```

Comment:
this makes it an generator, so it'll generate items only when the ES helper consumes them.